public_desktop/desktop-app.js

```diff
@@ -154,6 +154,47 @@ try {
 	ipc.on( 'is-calypso', function() {
 		ipc.send( 'is-calypso-response', document.getElementById( 'wpcom' ) !== null );
 	} );
+
+	ipc.on( 'app-config', function( event, config, debug, details ) {
+
+		// if this is the first run, show Windows and Mac users a pin app reminder
+		if ( details.firstRun ) {
```

Comment:
Should we verify that we're on the login page? I know that's the default but could change in the future?